aiogram_dialog/dialog.py

```diff
@@ -122,7 +122,9 @@ async def show(self, manager: DialogManager) -> None:
             )
         message = await manager.show(new_message)
         manager.current_stack().last_message_id = message.message_id
-        manager.current_stack().last_media_id = get_media_id(message)
+        media_id = get_media_id(message)
```

Comment:
get media_id can return None

so next calls to  media_id.file_id / media_id.file_unique_id can fail